cmd/istioctl/main.go

```diff
@@ -55,6 +61,38 @@ var (
 	key    model.Key
 	schema model.ProtoSchema
 
+	rootCmd = &cobra.Command{
+		Use:   "istioctl",
+		Short: "Istio control interface",
+		Long: fmt.Sprintf("Istio configuration command line utility. Available configuration types: %v",
+			model.IstioConfig.Kinds()),
+		PersistentPreRunE: func(*cobra.Command, []string) (err error) {
+			if kubeconfig == "" {
+				if os.Getenv("KUBECONFIG") != "" {
+					glog.V(2).Infof("Setting configuration from KUBECONFIG environment variable")
+					kubeconfig = os.Getenv("KUBECONFIG")
+				}
+			}
+
+			var client *kube.Client
```

Comment:
I'm curious, does the following give a linter warning about variable shadowing? It would seem cleaner and is allowed by spec:

```
client, err := kube.NewClient(...)
```